raaghu-elements/src/rds-stepper/rds-stepper.tsx

```
@@ -1,405 +1,451 @@
+              onClick={increasePageCountHandler}
+            >
+              Next
+            </button>
+          </div>
+        </>
+      )}
+
+      {props.stepperType == "advance" && (
+        <>
+          {props.advanceList?.map((e) => (
+            <div className={props.stepperSectionClass}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)